spec/src/main/asciidoc/java-api.adoc

```diff
@@ -145,6 +145,8 @@ public class CheckDiskspace implements HealthCheck {
 }
 ```
 
+`HealthCheckResponse` also provides default constructors to allow instantiation on the consuming end.
```

Comment:
```suggestion
`HealthCheckResponse` also provides several constructors to allow instantiation on the consuming end.
```

Reply:
the idea is to mention that there is a constructor with param to allow instantiation by 3rd party framework on consuming end